hooks/02-spread-support.chroot

```diff
@@ -1,39 +0,0 @@
-#!/bin/sh -x
```

Comment:
While understandably we cannot have this in the image anymore we need to ensure that subsequent core builds _can_ be tested. Let's see how we can re-package this in tests to achieve the same result.